### ISSUE TITLE
Fade out example does not work

### ISSUE THREAD

**Issue by bcowgill** [bentatum/react-fade#3]
The fade out example does not work in react 15.1.0

When I try the example as is, clicking the fade out button slowly fades out the content but when done it immediately becomes visible again. The setTimeout never appears to get called.

Clicking the button again, immediately hides the content and fires the timeout forever. This would be bad in a real app.

**Comment by episodeyang** [comment]
Also doesn't work with server-side rendering.

**Comment by Wasseratem** [comment]
True, its not good... no work in 2 years on this... pls delete this repo...
